joint_names:
- "joint_1"
- "joint_2"
- "joint_3"
- "joint_3"
- "joint_4"
- "joint_5"
first_frame: "base"
last_frame: "flange"

rest_time: 5 
positions:
- [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
- [0.0, 0.0, 0.0, 0.0, 0.0, 1.57]
- [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
- [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
- [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

